### PR TITLE
added linting action and fixed linting errors

### PR DESCRIPTION
**Pull request by Jurj-Bogdan**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### Description
Added a separate GitHub workflow to lint all markdown files not covered by the laminas-ci workflow, as discussed on [issue 222](https://github.com/laminas/getlaminas.org/issues/222)
Also fixed the linting errors i found, using the default config and the existing `.markdownlint,json` file.

### CODE REVIEW

File: data/advisories/LP-2020-01.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "LP-2020-01: XSS vectors in laminas-api-tools/api-tools"
 date: '2020-04-01T16:30:00-05:00'
 ---
 
-# LP-2020-01: XSS vectors in laminas-api-tools/api-tools
+## LP-2020-01: XSS vectors in laminas-api-tools/api-tools
```
Comment (froschdesign) [inline data/advisories/LP-2020-01.md:7]: Why does this need to be changed? It must be the first headline: https://getlaminas.org/security/advisory/LP-2020-01

Comment (gsteel) [reply data/advisories/LP-2020-01.md:7]: So looks like configuring `front_matter_title` to `''` will solve the problem

**Comment by Jurj-Bogdan** [comment]
Updated the config file to match the desired format

File: .markdownlint.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
     "default": true,
-    "MD013": false
+    "MD013": false,
+    "MD025": {
+        "front_matter_title": ""
+    }
```
Comment (gsteel) [inline .markdownlint.json:6]: Can I suggest you duplicate the config from CI: https://github.com/laminas/laminas-continuous-integration-action/blob/1.41.x/setup/markdownlint/markdownlint.json with the addition of the front-matter fixes - just for the sake of consistency with everything else.

Feel free to disagree… For example, all the lists will need the indentation updating etc

Comment (Jurj-Bogdan) [reply .markdownlint.json]: It does indeed make sense to use the same rules - i did however keep the initial changes coming from MD014 (removed the dollar signs used before commands, which on click would be copied to the clipboard as well), but that can be reverted if you feel it makes it more obvious that they're console commands.

Comment (gsteel) [reply .markdownlint.json]: From what I can see, all code blocks have a language association such as ` ```bash `, so removing the prompt is a good improvement as far as I'm concerned 👍

**Review by gsteel — APPROVED** [review]
LGTM @Jurj-Bogdan 👍 I'll leave it to @froschdesign for the final word :)

File: LICENSE.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 Laminas Project a Series of LF Projects, LLC.
+# Copyright (c) 2020 Laminas Project a Series of LF Projects, LLC
```
Comment (froschdesign) [inline LICENSE.md:1]: The same here.

File: README.md [hunk 4]
```diff
@@ -30,28 +30,28 @@ RELEASE_FEED_TOKEN=aaaabbbbccccddddeeeeffffgggg0000
 Second, install dependencies:
 
 ```bash
-$ composer install
+composer install
```
Comment (froschdesign) [inline README.md:33]: This does not match the deactivated rule number 14. This also applies to all other CLI commands.

https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md014.md

**Review by froschdesign — CHANGES_REQUESTED** [review]

**Review by froschdesign — APPROVED** [review]

**Pull request merged**